app/logic/Deployments.scala

```diff
@@ -57,4 +64,17 @@ object Deployments {
       event.note,
       event.result.getOrElse(Succeeded)
     )
+
+  def buildLinks(deployment: Deployment, jiraResponse: Option[WSResponse]) = Kleisli[Future, JIRA.Context, Seq[Link]] {
```

Comment:
is it ok to just return the deployment with the updated links or would you create a whole new Deployment object?